tests/test_image_utils.py

```diff
@@ -264,7 +264,9 @@ def test_normalize_image(self):
 
         # During the conversion rescale and channel first will be applied.
         expected = array.transpose(2, 0, 1).astype(np.float32) / 255.0
-        expected = (expected - np.array(mean)[:, None, None]) / np.array(std)[:, None, None]
+        expected = (expected - np.array(mean).astype(np.float32)[:, None, None]) / np.array(std).astype(np.float32)[
+            :, None, None
+        ]
```

Comment:
This is starting to be a less unreadable. Can we stored `np_mean = np.array(mean).astype(np.float32)[:, None, None])` and `np_std = np.array(std).astype(np.float32)[:, None, None]` on the lines above?